gmm.py

```diff
@@ -188,6 +212,41 @@ def score_samples(self, x):
         return score
 
 
+    def _cal_mutmal_x_cov(self, mat_a, mat_b):
```

Comment:
Looks to me as if this could be moved to an extra file (e.g. `helpers.py`) to improve readability, or what's your opinion?

Good to add these in any case for the extra stability! 👍

Reply:
I also don't think it should be a function of a class, later i'll modify.